ush/python/pygfs/task/archive.py

```diff
@@ -374,7 +374,7 @@ def _rename_cyclone_expt(arch_dict) -> None:
         if len(arch_dict.PSLOT) > 4:
             pslot4 = arch_dict.PSLOT[0:4].upper()
 
-        track_dir = arch_dict.COM_ATMOS_TRACK
+        track_dir = arch_dict.COMOUT_ATMOS_TRACK
```

Comment:
The lines following this will need to be updated to use the appropriate track_dir (even if they are both the same right now).

```suggestion
        track_dir_in = arch_dict.COMIN_ATMOS_TRACK
        track_dir_out = arch_dict.COMOUT_ATMOS_TRACK
```